chia/cmds/cmds_util.py

```diff
@@ -0,0 +1,12 @@
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.mempool_submission_status import MempoolSubmissionStatus
+from chia.wallet.transaction_record import TransactionRecord
+
+
+def transaction_submitted_msg(tx: TransactionRecord) -> str:
```

Comment:
I prefer this type of function to return string, rather than doing the printing itself.